packages/cubejs-api-gateway/src/gateway.ts

```diff
@@ -186,7 +186,8 @@ type BaseRequest = {
 
 type QueryRequest = BaseRequest & {
   query: Record<string, any> | Record<string, any>[];
-  queryType?: 'multi'
+  queryType?: 'multi';
+  apiType?: 'sql' | 'graphql' | 'rest';
```

Comment:
what about WebSocket server?) cC @RusovDmitriy Probably it will be cool to show it in APM.

Reply:
@ovr  Thanks! Let's add WS too.

@wolfeeeg  See https://github.com/cube-js/cube.js/blob/master/packages/cubejs-api-gateway/src/SubscriptionServer.ts#L108-L124